src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -166,6 +167,20 @@ public static LocalDate parseDate(String date) throws ParseException {
         return getDate(trimmedDate);
     }
 
+    /**
+     * Parses a {@code String date} into an {@code LocalDate}.
+     *
+     * @throws ParseException if date format is invalid.
+     */
+    public static LocalDate parseAnyDate(String date) throws ParseException {
```

Comment:
This should replace `parseDate`. Parser shouldn't be checking if the day is before today actually, it's a parser so any date that is valid should pass.

Reply:
okay, check today on wards at command side